molecule/testinfra/staging/app-code/test_securedrop_app_code.py

```diff
@@ -34,18 +35,6 @@ def test_securedrop_application_apt_dependencies(Package, package):
     assert Package(package).is_installed
 
 
-def test_securedrop_application_test_locale(File, Sudo):
-    """
-    Ensure SecureDrop DEFAULT_LOCALE is present.
-    """
-    securedrop_config = File("{}/config.py".format(
-        securedrop_test_vars.securedrop_code))
-    with Sudo():
-        assert securedrop_config.is_file
-        assert securedrop_config.contains("^DEFAULT_LOCALE")
-        assert securedrop_config.content.count("DEFAULT_LOCALE") == 1
```

Comment:
I don't think we need these tests because the new `Config` class will gracefully handle the missing value and set it to `en_US`.

Reply:
>I don't think we need these tests because the new Config class will gracefully handle the missing value and set it to en_US.

Correct, but, please make sure that is there is a test which checks for this default language value.